### ISSUE TITLE
Tags mode needs a refresh command

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ijt/emacs-nav#37]
```
...

```

Original issue reported on code.google.com by `issac.tr...@gmail.com` on 12 Aug 2009 at 7:29

**Comment by GoogleCodeExporter** [comment]
```
You can refresh the tag positions but adding new tags to a open buffer isn't 
working. 

(defun nav-tags-refresh ()
  (interactive)
  (goto-line 1)
  (nav-bufs-expand-tags))

I tried setting the alist nil still no avail. iMenu must need a new command to
rebuild the list.
```

Original comment by `matthew....@gmail.com` on 13 Aug 2009 at 6:39

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 8eaf49320e.
```

Original comment by `issac.tr...@gmail.com` on 14 Aug 2009 at 6:57
- Changed state: **Fixed**
